.stickler.yml

```diff
@@ -0,0 +1,10 @@
+linters:
+ flake8:
+ eslint:
+   config: "stylelint.config.js"
```

Comment:
Please also fix the linter errors. You seem to be using the wrong yml file hence the errors. Here is a link to the correct one https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml